packages/hash/frontend/src/blocks/page/BlockPortals.tsx

```diff
@@ -8,16 +8,25 @@ export interface PortalProps {
   portals: [HTMLElement, BlockPortal][];
 }
 
+/**
+ * Creates portals to render the elements that make up a specific block on the page, and provides shared context to both.
+ * The two elements rendered into portals are defined in BlockView (context controls) and ComponentView (block content).
+ * @param draftId the draftId of the block these portals belong to
+ * @param portals the pairings of nodes and elements needed to create the portals
+ */
 export const BlockPortals = ({ draftId, portals }: PortalProps) => {
```

Comment:
This appears to be the place where data can be shared between the block handle and the component view – I didn't want to do this in ProseMirror as I don't think this need involve PM.

If anyone knows a better way of sharing/triggering stuff between the handle and block content then please let me know.